### DISCUSSION TITLE
Problem Maintaining Login Session

### DISCUSSION THREAD

**Discussion by rahulgothwal5** [mozilla-mobile/android-components#11154] [General]
Hi everyone ,

Is there any reason why the Mozilla components for android doesn't maintain login sessions for certain websites like https://code.org/. Every time I open the website after opening the app I have to do a login.

**Comment by pocmo** [comment]
We do keep cookies. But it depends on what you are doing. Are you building a custom app or building on top of one of our apps? Which of the engine components are you using?
